modules/metrics/src/main/java/com/spotify/apollo/metrics/semantic/SemanticServiceMetrics.java

```diff
@@ -203,6 +207,18 @@ protected Ratio getRatio() {
         });
   }
 
+  private Supplier<Ratio> errorRatioSupplier(Supplier<Double> errorRateSupplier,
+                                             Supplier<Double> replyRateSupplier) {
+    // If a service is drained of traffic (for instance by moving traffic to another host or site),
+    // the meter values will decay exponentially towards zero. If left for some time (hours),
+    // rounding errors will cause the values of the meters to approach each other, thus skewing
```

Comment:
Exactly. In most cases it makes a jump from almost zero to almost 1. Has woken up people several times already. Do you want me to clarify the comment even more?